build/config/compiler/BUILD.gn

```diff
@@ -1774,14 +1804,38 @@ config("chromium_code") {
       # Warn on missing break statements at the end of switch cases.
       # For intentional fallthrough, use [[fallthrough]].
       "-Wimplicit-fallthrough",
-
-      # Warn on unnecessary extra semicolons outside of function definitions.
-      "-Wextra-semi",
     ]
 
+    if (!is_starboard) {
+      cflags += [
+        # Warn on unnecessary extra semicolons outside of function definitions.
+        "-Wextra-semi",
+      ]
+    }
+
+    # Suppress warning in old //net.
+    if (is_starboard) {
+      cflags += [
```

Comment:
Why do we need this change? Are the warnings caused by this build/config change?

Reply:
Considering that this seems to be specifically for the old //net, let's add a TODO here to come back and try to remove them later